kaiax/gasless/config.go

```diff
@@ -43,12 +43,15 @@ type GaslessConfig struct {
 }
 
 func GetGaslessConfig(ctx *cli.Context) *GaslessConfig {
-	allowedTokens := []common.Address{}
+	var allowedTokens []common.Address
 	for _, addr := range ctx.StringSlice(AllowedTokensFlag.Name) {
 		if addr == "all" {
 			allowedTokens = nil
 			break
 		}
+		if allowedTokens != nil {
+			allowedTokens = []common.Address{}
```

Comment:
did you mean this?
```
	if allowedTokens == nil {
		allowedTokens = []common.Address{}
	}
```
What was wrong with the previous logic?

Reply:
Oh, your code is right.

If no flag, this used to set empty slice to allowedTokens. But this was wrong. In that case, allowedTokens should be nil.